upgrade_guide/migrating_from_astyanax/language_level_changes/README.md

```diff
@@ -0,0 +1,66 @@
+             |                                                                                            |
+One Thrift   |                                                                                           -/
+COLUMNFAMILY |          
+             |  
+             |  /------------\              /---------------+---------------+---------+
+             |  |            |              |     Col 1     |     Col 4     |         |
+             |  | Key : 2    | ---------->  +---------------+---------------+ ...     |
+             |  |            |              |     Val : 'a' |     Val : 'b' |         |
+             |  \------------/              \---------------+---------------+---------+
+             |  
+             \- 
+
+
+                      -----------------------------------------------------------------------
+
+
+                                                     CQL
+                                                     
+             /-                              
+             |                               
+             |  /--------------------+---------------------------+-----------------------+----------------------+-----------------------------\
+             |  |       key          |           Col1            |         Col2          |         Col3         |            Col4             |  
+             |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+ -\
+             |  | cRED   1           |   c1AB     'a'            | cFA0     'b'          |          'c'         |             null            |  +--> One CQL
+   One CQL   |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+ -/    ROW
+   TABLE     |  |        2           |             'a'           |          null         |          null        |             'b'             |
+             |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+
+             |  |       ...          |            ...            |          ...          |          ...         |             ...             |
+             |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+
+             \-
+```
+
+Some of the columns of a *CQL Table* have a special role that is specifically
+related to the *Cassandra* architecture. Indeed, the *Row key* of the *Thrift Row*,
+becomes the *Partition Key* in the *CQL Table*, and can be composed of 1 or multiple
+*CQL columns* (the key column in Figure 1). The *“Column”* part of the Column-value
+component in a *Thrift Row*, becomes the *Clustering ColumnKey* in *CQL*, and can
+also be composed of multiple columns (in the figure, column1 is the only column 
+composing the *Clustering ColumnKey*).
```

Comment:
In the thrift example, are "col1, col2, col3" the cell names, and "a, b, c" the cell values? If so, I don't agree with the second part of the schema, cell names should translate to different values of the clustering column.
See the "Dynamic Column family" example in [this blog post](http://www.datastax.com/dev/blog/thrift-to-cql3): the thrift cell names (timestamps) become the *values* of the "time" CQL column. The text says
> map [...] the second component \[of the CQL primary key, i.e. the clustering column] (time) to the internal cell name. And the last CQL3 column (url) will be mapped to the cell value"

It gets more complicated with multiple clustering columns and multiple non-primary key rows (see the "Composite" and "Non-compact tables" examples respectively), but the *value* of the CQL clustering column always goes into the *name* of the internal cell.

I think the example should also illustrate a case where a single Thrift wide row translates to multiple CQL rows, it seems like that's a common hurdle for people coming from Thrift.